tests/utils_test.py

```diff
@@ -260,6 +260,42 @@ def test_combine_merge_values(self):
 
         self.assertEqual(actual_combine, expected_combine)
 
+    def test_convert_colors_to_hex_value(self):
+        color = {"red": 1, "green": 0.5, "blue": 0}
+        expected_hex = "#FF8000"
+
+        # successful convert from colors
+        hex = utils.convert_colors_to_hex_value(**color)
+        self.assertEqual(hex, expected_hex)
+
+        # successful convert from partial input
+        hex = utils.convert_colors_to_hex_value(green=1)
+        self.assertEqual(hex, "#00FF00")
+
+        # throw ValueError on color values out of range (0-1)
+        with self.assertRaises(ValueError):
+            utils.convert_colors_to_hex_value(1.23, 0, -50)
+
+    def test_convert_hex_to_color(self):
+        hex = "#FF8000"
+        expected_color = {"red": 1, "green": 128 / 255, "blue": 0}
```

Comment:
for the green value please use some random value with some decimals, just in case and not some rounded value if possible, something like: `0.49803922` which is the return value for the green field of one of my sheets.